api/v1alpha1/openapi.yaml

```diff
@@ -2264,24 +2311,33 @@ components:
       properties:
         creationTimestamp:
           type: string
+          format: date-time
         deletionTimestamp:
           type: string
+          format: date-time
         name:
           type: string
+          minLength: 1
+          maxLength: 63
+          pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
           description: name of the object
-        labels:
+        labels: # Length and pattern will be validated by API server
           type: object
           additionalProperties:
             type: string
           description: 'Map of string keys and values that can be used to organize and categorize (scope and select) objects.'
         generation:
           type: integer
           format: int64
+          minimum: 0
           description: A sequence number representing a specific generation of the desired state. Populated by the system. Read-only.
         owner:
           type: string
+          minLength: 0
+          maxLength: 127
+          pattern: '^([a-z0-9]([-a-z0-9]*[a-z0-9])?\/)?[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
```

Comment:
This pattern is not correct. As implied by the description, it prefixes the value with the `Kind`.

I think this relates to previous comments from @avishayt , should we maybe only restrict the input field? The owner is being set by the system,  and should not be possible to overwrite it from the clients.

With the current pattern, any PUT request on a device owned by a fleet will fail.

Reply:
I've removed this validation and will add a fixed version in a follow-up PR.